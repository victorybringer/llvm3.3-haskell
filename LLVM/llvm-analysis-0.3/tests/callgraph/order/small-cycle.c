void h()
{
}

void g();

void f()
{
  g();
  h();
}

void g()
{
  f();
}

int main(int argc, char *argv[])
{
  g();
  return 0;
}
