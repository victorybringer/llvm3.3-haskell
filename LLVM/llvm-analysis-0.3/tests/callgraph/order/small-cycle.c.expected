[fromList ["main"], fromList ["g", "f"], fromList ["h"]]
